### ISSUE TITLE
NotificationJobService error with today 2018-12-10 build

### ISSUE THREAD

**Issue by diotto** [AmazMod/AmazMod#329]
Receiving this error everytime i open app on phone:
```
2018-12-10 00:37:55.420 17315-17315/com.edotassi.amazmod E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.edotassi.amazmod, PID: 17315
    java.lang.NullPointerException: Attempt to invoke virtual method 'com.huami.watch.transport.DataBundle amazmod.com.transport.data.NotificationData.toDataBundle(com.huami.watch.transport.DataBundle)' on a null object reference
        at com.edotassi.amazmod.notification.NotificationJobService.processCustomNotificationPosted(NotificationJobService.java:279)
        at com.edotassi.amazmod.notification.NotificationJobService.access$100(NotificationJobService.java:28)
        at com.edotassi.amazmod.notification.NotificationJobService$1.run(NotificationJobService.java:105)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6123)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:867)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:757)
2018-12-10 00:37:55.870 17315-17410/com.edotassi.amazmod E/Fabric: Settings request failed.
2018-12-10 00:37:55.899 17315-17408/com.edotassi.amazmod I/FA: Tag Manager is not found and thus will not be used
2018-12-10 00:37:55.916 17315-17408/com.edotassi.amazmod D/FA: Logging event (FE): app_exception(_ae), Bundle[{firebase_event_origin(_o)=clx, _r=1, timestamp=1544409471378, fatal=1}]
2018-12-10 00:37:56.331 17315-17410/com.edotassi.amazmod E/Fabric: Error performing auto configuration.
2018-12-10 00:37:56.333 17315-17438/com.edotassi.amazmod E/CrashlyticsCore: Error occurred sending report com.crashlytics.android.core.SessionReport@6c16a1e
2018-12-10 00:37:56.340 17315-17438/com.edotassi.amazmod E/CrashlyticsCore: Error occurred sending report com.crashlytics.android.core.SessionReport@fe382ff
2018-12-10 00:37:56.343 17315-17438/com.edotassi.amazmod E/CrashlyticsCore: Error occurred sending report com.crashlytics.android.core.SessionReport@602a8cc
2018-12-10 00:37:56.348 17315-17438/com.edotassi.amazmod E/CrashlyticsCore: Error occurred sending report com.crashlytics.android.core.SessionReport@8583715
```

**Comment by diotto** [comment]
I'm testing if its some trash in my phone. Removed all app data in phone (and watch) and rebooted both.

**Comment by lfom** [comment]
It was due to "ghost" notifications entries in the store, it should have been fixed with one of my last commits (68bc8e2908c8fda3ae976d8a6683f04b4759fdf3)...
